source/slang/slang-session.cpp

```
@@ -59,6 +59,14 @@ Linkage::Linkage(Session* session, ASTBuilder* astBuilder, Linkage* builtinLinka
     , m_cmdLineContext(new CommandLineContext())
     , m_stringSlicePool(StringSlicePool::Style::Default)
 {
+    // [TEST] Deliberate signed integer overflow to verify sanitizer CI catches it.
+    // UBSan will flag this; ASan will not. Remove after verification.
+    {
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)